motoman_driver/src/io_relay.cpp

```diff
@@ -113,12 +114,20 @@ bool MotomanIORelay::readSingleIoCB(
 
   if (!result)
   {
-    ROS_ERROR_NAMED("io.read", "Reading IO element %d failed", req.address);
-    return false;
+    std::stringstream message;
+    message << "Reading IO element " << req.address << " failed";
```

Comment:
Please update the messages here with a single quote or parentheses around the streamed variables like `req.address`. In the case that a user specifies a blank address, this error message would be somewhat confusing and unhelpful: `Reading IO element  failed` vs `Reading IO element () failed`

Reply:
Isn't `address` an integer?

Reply:
```suggestion
    message << "Reading IO element (" << req.address << ") failed";
```

Reply:
@marip8: if `address` is an integer, can it ever be empty?

Reply:
Good point; I would think it probably couldn't be empty (opposed to strings that are blank by default). This change is probably superfluous then

Reply:
I fully agree with you in case this was a `string` btw. Nothing more perplexing than seeing weird error messages.